core/assets/bundles/bundle_vi.properties

```diff
@@ -1050,8 +1050,8 @@ unit.corvus.name = Corvus
 block.resupply-point.name = Điểm tiếp tế
 block.parallax.name = Parallax
 block.cliff.name = Vách đá
-block.sand-boulder.name = Tường cát
-block.basalt-boulder.name = Tường đá basalt
+block.sand-boulder.name = Tường cát đá
+block.basalt-boulder.name = Tường đá huyền vũ nham thạch
```

Comment:
Magical lava rock wall?

Reply:
it's not even magical
let nglam understand it

Reply:
you cant just google translate the whole thing , because the translation is going to make the meaning into a mess of a bundle

Reply:
Oh ok